.github/scripts/js/ci.js

```diff
@@ -896,6 +907,16 @@ module.exports.runWorkflowForPullRequest = async ({ github, context, core, ref }
     core.endGroup();
   }
 
+  if (command.setE2eShouldSkipped !== null) {
```

Comment:
```suggestion
  if (command.setE2eShouldSkipped) {
```
That's enough

Reply:
No. This value stores [3 states](https://github.com/deckhouse/deckhouse/blob/1ee6bfd3ce780c58740406052db6c213b02f331a/.github/scripts/js/ci.js#L851-L854) "skip label was not set (do nothing)" it is null, "skip label added or removed" (true/false).